source/process/spec-process.rst

```diff
@@ -0,0 +1,46 @@
+Spec Review Process
+===================
+
+This is a rough guideline for the process for drafting and reviewing specs. It’s meant to be used as a starting point, but not all specs will follow this exact process - please use your judgment, as it’s something that will vary depending on the situation.
+
+1. Brainstorm / research.
+    - Research topic
+    - Research customer requirements
+    - Pair design brainstorm
+    - User workflows
+    - Consider options / vision for UI
+2. Initial draft.
+    - Goal / “Why"
+
+      - Link to customer data / feedback
+      - Link back to company objectives
+    - User Stories
+    - Rough and Quick Wireframes / Mock ups
+
+      - Flows of screens
+      - Focus should be on the mock ups, with text that supports design
+
+    - Questions
+    - “Not in scope”
+    - Success Metrics
+    - Pair reviews
+
+      - Iterate on designs as needed
+      - Early check in with developers to identify any potential issues
+3. Share with team.
+    - Post draft with @channel in Spec Review channel, asking to review and add comments.
+4. Spec review.
+    - Schedule a meeting with interested people to discuss comments and any other questions about the spec:
+
+      - UX Design
```

Comment:
I was using "UX Design" and "Test" more as someone from the UX Design / test teams as opposed to individuals, but we should be consistent on the list.